src/base/types/MATModel.jl

```diff
@@ -244,3 +247,19 @@ function Base.convert(::Type{MATModel}, m::MetabolicModel)
         ),
     )
 end
+
+"""
+    reaction_name(m::MATModel, mid::String)
+
+Extract metabolite compartment from `rxnNames`.
+"""
+reaction_name(m::MATModel, rid::String) =
+    _maybemap(x -> x[findfirst(==(rid), reactions(m))], get(m.mat, "rxnNames", nothing))
+
+"""
+    metabolite_name(m::MATModel, mid::String)
+
+Extract metabolite compartment from `metNames`.
+"""
+metabolite_name(m::MATModel, mid::String) =
+    _maybemap(x -> x[findfirst(==(mid), metabolites(m))], get(m.mat, "metNames", nothing))
```

Comment:
do we have downloadable models for testing this?

Reply:
hmm yes, but I am not sure how stable the link is :/